expo/scheduler/functions.py

```diff
@@ -379,7 +380,7 @@ def get_default_range():
     return (today + Duration(hours=8), today + Duration(hours=28))
 
 
-def cal_times_for_conf(conference, day_name):
+def cal_times_for_conf(conference, day_name=None):
```

Comment:
not sure what's up here, but if i do 'day=Saturday' - I get Saturday Events AND Friday events.  And if I do 'day=Sunday' - I get Sunday Events and Friday events.